groups/postgres/groups.go

```diff
@@ -1183,12 +1184,16 @@ func toDBGroup(g groups.Group) (dbGroup, error) {
 	if g.UpdatedBy != "" {
 		updatedBy = &g.UpdatedBy
 	}
+	var desc sql.NullString
+	if g.Description.Set {
```

Comment:
Please change logic like given below 
```suggestion
	if g.Description != "" {
		desc = sql.NullString{String: g.Description.Value, Valid: true}
	}
```